SOLR-16228: Audit how we refer to Solr Modules (#889)
apache/solr: solr/solr-ref-guide/modules/indexing-guide/pages/language-analysis.adoc

== OpenNLP Integration

> The `lucene/analysis/opennlp` package provides OpenNLP integration via several analysis components: a tokenizer, a part-of-speech tagging filter, a phrase chunking filter, and a lemmatization filter.

Reviewer: ```suggestion
The Lucene-Module `lucene/analysis/opennlp` provides OpenNLP integration via several analysis components: a tokenizer, a part-of-speech tagging filter, a phrase chunking filter, and a lemmatization filter.
```

Author: I looked and see _Lucene module_ used, so going to change to that...

The dictionary resource must be named `path/to/_language_.dict` and have an associated `.info` metadata file.
See http://morfologik.blogspot.com/[the Morfologik project] for details.
If the dictionary attribute is not provided, the Polish dictionary is loaded and used by default.

== Analysis Extras Module

> Many of the language features listed above are supported by the Analysis Extras module.
> 
> This is provided via a xref:configuration-guide:solr-modules.adoc[Solr Module] that needs to be enabled before use.
> 
> The module name is `analysis-extras`.
> 
> Additional details on the specific jar files required is documented in the `modules/analysis-extras/README.md`.

Reviewer 2: ```suggestion
Many of the language features listed above are supported by the `analysis-extras` xref:configuration-guide:solr-modules.adoc[Solr Module] that needs to be enabled before use.

Additional details on the specific jar files required can be found in the https://github.com/apache/solr/blob/main/solr/modules/analysis-extras/README.md[Module's README].
```
